src/extension/module.js

```diff
@@ -1546,6 +1549,104 @@
     }
   }
 
+  function getTimeAgo(dateParam) {
+    if (!dateParam) return ' --- ';
+    const date = typeof dateParam === 'object' ? dateParam : new Date(dateParam);
+
+    const DAY_IN_MS = 86400000; // 24 * 60 * 60 * 1000
+    const today = new Date();
+    const yesterday = new Date(today - DAY_IN_MS);
```

Comment:
why do you have a constant for `DAY_IN_MS` but not for the `1000` below ? i.e. no need for the constant if you don't need it more than oncee.